### ISSUE TITLE
Configurable props for DialogTitle and DialogContent for @material-ui v4

### ISSUE THREAD

**Issue by imjordanxd** [jonatanklosko/material-ui-confirm#39]
Accessibility is important for dialogs. In this API, you cannot provide props to link the title and content. I cannot upgrade to Mui V5 yet as I have to support IE11.

**Comment by jonatanklosko** [comment]
The change from #38 is available in `v2.1.4` (compatible with mui v4).
